addons/website/static/src/scss/website.editor.ui.scss

```diff
@@ -1,813 +1,803 @@
-                color: $o-we-color-text-light;
+                color: $o-we-color-text-normal;
             }
-        }
 
-        &.active {
-            background-color: $o-we-color-dark;
-            color: $o-we-color-text-light;
+            &:hover {
+                color: $o-we-color-text-light;
 
-            >.fa {
+                >.fa {
+                    color: $o-we-color-text-light;
+                }
+            }
+
+            &.active {
+                background-color: $o-we-color-dark;
                 color: $o-we-color-text-light;
+
+                >.fa {
+                    color: $o-we-color-text-light;
+                }
             }
         }
-    }
 
-    >.btn-group {
-        margin-top: 0;
+        >.btn-group {
+            margin-top: 0;
 
-        &.show,
-        .btn-group.show {
-            .dropdown-toggle {
-                border-top: 2px solid white;
-                background-color: $o-we-color-darker;
-                color: white;
-                box-shadow: none;
-
-                >.fa,
-                >span {
+            &.show,
+            .btn-group.show {
+                .dropdown-toggle {
+                    border-top: 2px solid white;
+                    background-color: $o-we-color-darker;
                     color: white;
+                    box-shadow: none;
+
+                    >.fa,
+                    >span {
+                        color: white;
+                    }
                 }
             }
         }
-    }
 
-    .dropdown-menu {
-        margin: 0;
-        border-radius: 0;
-        background-color: $o-we-color-darker;
-        color: $o-we-color-text-normal;
-        white-space: normal;
-
-        .dropdown-item {
-            width: 100%;
-            color: $o-we-color-text-light;
+        .dropdown-menu {
+            margin: 0;
+            border-radius: 0;
+            background-color: $o-we-color-darker;
+            color: $o-we-color-text-normal;
+            white-space: normal;
 
-            >.fa {
+            .dropdown-item {
+                width: 100%;
                 color: $o-we-color-text-light;
-            }
-
-            &:hover {
-                background-color: $o-we-tab-active-bg;
-                color: white;
 
                 >.fa {
+                    color: $o-we-color-text-light;
+                }
+
+                &:hover {
+                    background-color: $o-we-tab-active-bg;
                     color: white;
+
+                    >.fa {
+                        color: white;
+                    }
                 }
             }
         }
-    }
 
-    .note-para .dropdown-menu {
-        min-width: 147px;
-    }
+        .note-para .dropdown-menu {
+            min-width: 147px;
+        }
 
-    .note-style .dropdown-item * {
-        color: $o-we-color-text-light;
+        .note-style .dropdown-item * {
+            color: $o-we-color-text-light;
+        }
     }
-}
+}
```

Comment:
Could you leave the new line at the end of the file ? Not adding them in a new file is a thing but having an editor which removes them in existing files... :cry: